pegasus/sites.v3/hourofcode.com/views/hoc_events_map_replacement.haml

```diff
@@ -1,5 +1,7 @@
 %link{rel: "stylesheet", type: "text/css", href: "/css/map.css"}
-%script{src: minifiable_asset_path("js/hourofcode.com/views/hoc_events_map_replacement.js")}
+%link{href:'https://api.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css', rel:'stylesheet'}
```

Comment:
Why did you consolidate the scripts and links related to mapbox into one haml file?

Reply:
Cleanliness (and yes). It's much easier to maintain these scripts and styles in the file that actually uses them rather than scattered around in the haml files that use this template.